charts/catalog/templates/crds.yaml

```diff
@@ -0,0 +1,608 @@
+              clusterServiceBrokerName:
+                type: string
+              externalName:
+                type: string
+                maxLength: 63
+              externalID:
+                type: string
+                maxLength: 63
+              description:
+                type: string
+              bindable:
+                type: boolean
+              bindingRetrievable:
+                type: boolean
+              planUpdatable:
+                type: boolean
+              tags:
+                type: array
+                items:
+                  type: string
+              requires:
+                type: array
+                items:
+                  type: string
+              externalMetadata:
+                type: object
+              defaultProvisionParameters:
+                type: object
+---
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: serviceplans.servicecatalog.k8s.io
+  annotations:
+    "helm.sh/hook": crd-install
+spec:
+  group: servicecatalog.k8s.io
+  version: v1beta1
+  scope: Namespaced
+  names:
+    plural: serviceplans
+    singular: serviceplan
+    kind: ServicePlan
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          required:
+          - "serviceClassRef"
+          - "serviceBrokerName"
+          - "externalName"
+          - "externalID"
+          - "description"
+          properties:
+            serviceClassRef:
+              type: object
+              required:
+              - "name"
+              properties:
+                name:
+                  type: string
+                  maxLength: 63
+            serviceBrokerName:
+              type: string
+            externalName:
+              type: string
+              maxLength: 63
+            externalID:
+              type: string
+              maxLength: 63
+            description:
+              type: string
+            bindable:
+              type: boolean
+            free:
+              type: boolean
+            instanceCreateParameterSchema:
+              type: object
+            instanceUpdateParameterSchema:
+              type: object
+            serviceBindingCreateParameterSchema:
+              type: object
+            serviceBindingCreateResponseSchema:
+              type: object
+            defaultProvisionParameters:
+              type: object
+
+---
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterserviceplans.servicecatalog.k8s.io
+  annotations:
+    "helm.sh/hook": crd-install
+spec:
+  group: servicecatalog.k8s.io
+  version: v1beta1
+  scope: Cluster
+  names:
+    plural: clusterserviceplans
+    singular: clusterserviceplan
+    kind: ClusterServicePlan
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          required:
+          - "clusterServiceClassRef"
```

Comment:
why it's required?

Reply:
https://github.com/kubernetes-incubator/service-catalog/blob/master/pkg/apis/servicecatalog/validation/serviceplan.go#L110

Reply:
ok, so let's merge it like this and in https://github.com/kyma-project/kyma/issues/2792 we will recheck what kind of validation choose based on user experience :)